use-api.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: docs
+title:  "How to Use the API"
+---
+
+## Access methods 
+The API is public and available for GET requests at [https://api.rockarch.org](https://api.rockarch.org). 
+
+Get started with your favorite API tool or script. We do not require an API key.
+
+- If you are new to scripting or working with APIs, consider using a tool like [Hoppscotch](https://hoppscotch.io/) or [Postman](https://www.postman.com/).
+- Browse the API online at [https://api.rockarch.org](https://api.rockarch.org).
+- We provide an [API client](https://pypi.org/project/rac-api-client/) to simplify requests if you are writing Python scripts.
+
+
+## Quick start 
+Data is accessed through GET requests using API endpoints. For a list of available endpoints, see the [endpoints and parameters](#endpoints-and-parameters) section of this document.
+
+Example 1:
+Use the `/collections` endpoint to get a list of all of our archival collections, which are intellectually significant groups of records:
+
+```
+GET https://api.rockarch.org/collections
+```
+
+Example 2:
+Get data about one specific collection, replacing {id} with the collection's identifier (example id: `H45i6yf7MUHuaRwQVupvg5`):
+
+```
+GET https://api.rockarch.org/collections/{id}
+```
+See what a response looks like in the browseable API by opening the link in your browser: [https://api.rockarch.org/collections/H45i6yf7MUHuaRwQVupvg5](https://api.rockarch.org/collections/H45i6yf7MUHuaRwQVupvg5).
+
+Example 3:
+Use the `/objects` endpoint and `online` parameter to get data for all archival objects that have digital versions available. Objects are defined as intellectually significant groups of records in a collection that do not have children:
+
+```
+GET https://api.rockarch.org/objects?online=true
+```
+
+Note that `online` is a query parameter. By convention, these are included after a `?` in the URL. To add multiple query parameters, separate the parameters by `&`. For example, to use both the `online` and `start_date` query parameters:
+
+```
+https://api.rockarch.org/objects?online=true&start_date=1950
+```
+
+## Understanding the data that comes back 
+We provide [JSON](https://www.json.org/json-en.html)-formatted data.
+
+Endpoints are paginated. We show 50 records per page by default. Pagination can be controlled via the following query parameters:
+ - `limit` to set how many records each page will return
+ - `offset` to request a specific page of results.
+
+Example: get the **second** page of agent records consisting of **10** records:
+
+```
+GET https://api.rockarch.org/agents?limit=10&offset=10
+```
+
+## Endpoints and parameters
```

Comment:
I also think it could be helpful if this section was a separate page. Approaching it as someone who is not as familiar working with APIs - I think it would help with the flow of the documentation. The link in the Quick Start section would help readers get to it